src/senselab/audio/tasks/features_extraction/torchaudio.py

```diff
@@ -36,7 +37,10 @@ def extract_spectrogram_from_audios(
     )
     spectrograms = []
     for audio in audios:
```

Comment:
we should perhaps discuss how to parallelize such loops for large datasets/long recordings in general. let's make this a key part of our code review target.